specification/attestation/data-plane/Microsoft.Attestation/preview/2018-09-01-preview/examples/Get_MetadataConfiguration.json

```diff
@@ -0,0 +1,40 @@
+{
+  "parameters": {
+    "tenantBaseUrl": "'https://mytest.attest.azure.net'",
+    "api-version": "2018-09-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "response_types_supported": [
+          "token",
+          "none"
+        ],
+        "id_token_signing_alg_values_supported": [
+            "RS256"
+        ],
+        "revocation_endpoint": "http://localhost:8080/revoke",
+        "jwks_uri": "http://localhost:8080/certs",
```

Comment:
I don't think we'll ever return http://localhost:8080 in any production scenario.

Reply:
This is an example I am using, just change it to https://mytest.attest.azure.net/revoke and https://mytest.attest.azure.net/certs

Reply:
Is that what other azure APIs do for their APIs which return URLs? Use localhost based URLs in their examples? Or do they have some other convention for URLs enbedded in responses?

Reply:
In the swagger, it shows both Get "certs" and ".well-known/openid-configuration"APIs return a type of object for 200 response. To pass the swagger validation, I need to add additional example for each APIs that we defined in the swagger. The user doesn't know what type of object the API return. then  I just take the example from postman.

Reply:
The response for "certs" should be a RFC 2117 JSON Web Key, we have an existing swagger for that. The response for ".well-known/openid-configureation" is an OpenID metadata discovery document. There has to be a swagger for those in the repo.